homeassistant/components/weatherflow_cloud/sensor.py

```diff
@@ -67,6 +67,18 @@ class WeatherFlowCloudSensorEntityDescriptionWebsocketObservation(
     value_fn: Callable[[WebsocketObservation], StateType | datetime]
 
 
+def map_precip_type(value: int) -> str | None:
```

Comment:
we don't need this method, just move the map to a `PERCEPITATION_TYPE` const, and call `PERCEPITATION_TYPE.get(data.precip_analysis_type_yesterday)` directly in the lambda of the sensor description.